packages/application-shell/src/components/quick-access/quick-access.tsx

```diff
@@ -286,7 +286,13 @@ const QuickAccess = (props: Props) => {
           keywords: variantKey ? [variantKey] : undefined,
           action: {
             type: actionTypes.go,
-            to: `/${applicationContext.project.key}/products/${productId}/${variantId}`,
+            to: oneLineTrim`
```

Comment:
<img width="579" alt="CleanShot 2020-03-16 at 10 59 23@2x" src="https://user-images.githubusercontent.com/1877073/76744905-3ff3d380-6775-11ea-9e9e-eae225658574.png">

This should go to a specific variant? Isn't the path correct?

Reply:
there should be a `/variants/` between the product id an the variant id, was the only change i made

Reply:
Got it. Sorry.